.readthedocs.yaml

```diff
@@ -0,0 +1,15 @@
+version: "2"
+
+build:
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.12"
+  jobs:
+    post_install:
+      - curl -fsSL https://pixi.sh/install.sh | bash
+      - export PATH="$HOME/.pixi/bin:$PATH" && pixi install
+      - pixi shell -c "pixi install && sphinx-build -T -b html -d _build/doctrees -D language=en . $READTHEDOCS_OUTPUT/html"
```

Comment:
Btw `pixi install` is not needed, at most `pixi shell`

Reply:
> Unfortunately, RTD runs its own `sphinx-build` command (hence `pre_build`, `post_build`, not build itself). So just running `pixi shell` should suffice 🤞 If it doesn't, this may be a more tricky issue to solve...

Seems like it doesn't, maybe tricky? What if  the `pixi shell` an interactive shell which is not supported in the ReadTheDocs environment, maybe?

Reply:
https://readthedocs.org/projects/gluex-nstar/builds/24669635/